pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -705,10 +705,20 @@ func createBackupScheduleHook(
 	if !ok {
 		return nil, nil, nil, false, nil
 	}
+
 	eval, err := makeScheduledBackupEval(ctx, p, schedule)
 	if err != nil {
 		return nil, nil, nil, false, err
 	}
+	destinations, errDest := eval.destination()
+	if errDest != nil{
+		return nil, nil, nil, false, errDest
+	}
+	errExist := checkForExistingBackupsInCollection(
```

Comment:
So I don't think this is exactly what we want. `checkForExistingBackupsInCollection` will contact the cloud/local storage destination and check if a previous backup job has written its data there. This is useful when we are enforcing that a new backup job will not overwrite a previous backup.

This is slightly different from the goal of `CREATE SCHEDULE IF NOT EXISTS`. A schedule is responsible for spawning backup jobs. The `IF NOT EXISTS` clause on a schedule should not concern itself with the actual backup job that will be run. It is responsible for checking that we do not already have a schedule with the same label as this one. If we do find a pre-existing schedule with the same label, we want this to be a noop thereby providing idempotency to the `CREATE SCHEDULE` command.

To solve this, we would need to query the `system.scheduled_jobs` table to check if there is an entry for a schedule with the same label as `eval.ScheduleLabel`. Lets sync offline about the best way to do this, I'll be able to explain better in person.

Reply:
done